qtt/tests/test_zi_hdawg8.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+from unittest.mock import MagicMock, call
+
+from qtt.instrument_drivers.virtualAwg.awgs.ZurichInstrumentsHDAWG8 import ZurichInstrumentsHDAWG8
+from qtt.instrument_drivers.virtualAwg.awgs.common import AwgCommonError
+
+
+class TestZurichInstrumentsHDAWG8(unittest.TestCase):
```

Comment:
Missing test cases for `run`, `stop`, `delete_waveforms`, ... It does mean that coverage of the ZurichInstrumentsHDAWG8 will not be 100%.
Since this is in `qtt` I am fine with not doing 100%.